templates/log4j2.xml.erb

```diff
@@ -138,7 +138,7 @@ if ! httpAppenders.empty?
 <%- end -%>
         </Logger>
 <%- if @herp -%>
-        <Logger name="org.openrepose.herp.post.filter" level="info">
+        <Logger name="org.openrepose.herp.post.filter" level="info" additivity="false">
```

Comment:
Interesting... so thats just the repose behavior with it, ya? The definition of it is "Additivity is set to true by default, that is children inherit the appenders of their ancestors by default" which is why i asked the question.   Seems like we'd want children to inherit the appenders